src/pages/signing_official_console/SigningOfficialTable2.jsx

```diff
@@ -0,0 +1,286 @@
+    );
+  }, [researchers]);
+
+  useEffect(() => {
+    recalculateVisibleTable({
+      tableSize, pageCount,
+      filteredList: filteredResearchers,
+      currentPage,
+      setPageCount,
+      setCurrentPage,
+      setVisibleList: setVisibleResearchers
+    });
+  }, [tableSize, pageCount, filteredResearchers, currentPage]);
+
+  const goToPage = useCallback((value) => {
+    if (value >= 1 && value <= pageCount) {
+      setCurrentPage(value);
+    }
+  }, [pageCount]);
+
+  const changeTableSize = useCallback((value) => {
+    if (value > 0 && !isNaN(parseInt(value))) {
+      setTableSize(value);
+    }
+  }, []);
+
+  const paginationBar = (
+    <PaginationBar
+      pageCount={pageCount}
+      currentPage={currentPage}
+      tableSize={tableSize}
+      goToPage={goToPage}
+      changeTableSize={changeTableSize}
+    />
+  );
+
+  const processResearcherRowData = (researchers = []) => {
+    return researchers.map(researcher => {
+      const {displayName, /*count = 0,*/ roles, libraryCards} = researcher;
+      const libraryCard = !isEmpty(libraryCards) ? libraryCards[0] : {};
+      const email = researcher.email || libraryCard.userEmail;
+      const id = researcher.userId || email;
+      return [
+        displayNameCell(displayName, email, id),
+        ...dacs.map(dac => DAACell(dac, researcher, signingOfficial.institutionId, daas, refreshResearchers, setResearchers))
+      ];
+    });
+  };
+
+  return (
+    <>
+      <div style={{ display: 'flex', justifyContent: 'space-between', alignItems: 'flex-end', width: '112%', marginLeft: '-6%' }}>
+        <div style={Styles.LEFT_HEADER_SECTION}>
+          <div style={{ ...Styles.HEADER_CONTAINER, marginRight: '-7%' }}>
+            <div style={{ ...Styles.SUB_HEADER,
+              marginTop: '0',
+              fontFamily: 'Montserrat',
+              fontWeight: 600,
+              fontSize: '2.8rem'}}>
+              My Institution&apos;s Researchers
+            </div>
+            <div style={Object.assign({}, Styles.MEDIUM_DESCRIPTION, {
+              fontSize: '16px',
+            })}>
+              Issue, Update, or Deactivate for User&apos;s ability to request access to datasets, by agreeing to
+            </div>
+            <div style={Object.assign({}, Styles.MEDIUM_DESCRIPTION, {
+              fontSize: '16px',
+            })}>
+              Data Access Committee&apos;s (DAC&apos;s) Data Access Agreements (DAAs) in the table below.
+            </div>
+            <div style={Object.assign({}, Styles.MEDIUM_DESCRIPTION, {
+              fontSize: '16px',
+            })}>
+              Issuing a checkmark in a cell for a researcher denotes your approval of that researcher
+            </div>
+            <div style={Object.assign({}, Styles.MEDIUM_DESCRIPTION, {
+              fontSize: '16px',
+            })}>
+              to request data from the listed DAC, according to its linked DAA.
+            </div>
```

Comment:
Two comments here
1. This construction is repetitive. I think we can simplify this with a max width percentage instead of separate divs.
2. The content here doesn't mention Library Cards at all. Researchers will see the phrase Library Card on their profile page so SOs visiting this page may not realize that changes here will trigger changes to the LC section of their users' profile. I think we need some PO feedback on this language to ensure there is no confusion.

Reply:
This language had come directly from the mock which I think Lou got from Jonathan, but I can check with him to confirm. If it was not directly from Jonathan, I can reach out & confirm what we want to be saying here.

Reply:
Follow-up: Lou has set up a meeting with Jonathan & me for next Monday to discuss & clarify this language and the larger conversation of what "issuing a Library Card" now means for a Signing Official.